test/unit/model/validation.test.js

```diff
@@ -321,37 +317,14 @@ describe(Support.getTestDialectTeaser('InstanceValidator'), () => {
         });
 
         it('should allow decimal as scientific notation', async () => {
```

Comment:
According to the support table, we could enable support for floating points

| dialect    | int | float | decimal |
|------------|-----|-------|---------|
| mysql      | ❌   | ✅     | ❌       |
| mariadb    | ❌   | ✅     | ❌       |
| postgresql | ❌   | ✅     | ✅       |
| sqlite     | ✅   | ✅     | N/A     |
| mssql      | ❌   | ✅     | ❌       |
| db2        | ❌   | ✅     | ❌       |
| ibmi       | ?   | ?     | ?       |
| snowflake  | ?   | ?     | ?       |

Can't enable it for decimal or ints because we'd need to parse the value as a floating point first and they'd lose precision